docs/shared/packages/powerpack-owners/powerpack-owners-plugin.md

```diff
@@ -1,6 +1,326 @@
 ---
 title: Overview of the Nx powerpack-owners Plugin
-description: The Nx Plugin for esbuild contains executors and generators that support building applications using esbuild. This page also explains how to configure esbuild on your Nx workspace.
+description: The Nx Powerpack Owners plugin provides the ability to define code ownership based on projects in addition to files
 ---
 
-Powerpack owners
+The `@nx/powerpack-owners` plugin extends the CODEOWNERS functionality to allow you to define code ownership based on projects in addition to the standard file-based definitions. It leverages the [`nx sync`](/concepts/sync-generators) command to compile `owners` configuration settings from `nx.json` and project configuration files into valid CODEOWNERS files for [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners), [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/set-up-and-use-code-owners/) or [GitLab](https://docs.gitlab.com/ee/user/project/codeowners/).
+
+With this plugin, you can specify code ownership using the same project glob syntax as [`nx run-many`](/nx-api/nx/documents/run-many#examples). This allows you to easily define rules for multiple projects that may not be located in the same directory. Also, the CODEOWNERS rules will not need to be revisited if a project location is changed or a new project is added.
+
+{% callout title="This plugin requires an active Nx Powerpack license" %}
+In order to use `@nx/powerpack-owners`, you need to have an active Powerpack license. If you don't have a license or it has expired, the syncing process will stop working and you'll need to manually maintain your CODEOWNERS file.
+{% /callout %}
+
+## Setup
+
+1. [Activate Powerpack](/recipes/installation/activate-powerpack) if you haven't already
+2. Install the package
+
+   ```shell
+   nx add @nx/powerpack-owners
+   ```
+
+3. Configure Ownership
+
+   Configure the `@nx/powerpack-owners` plugin in the `nx.json` file or in individual project configuration files. Consult the [Owners Configuration Reference](#owners-configuration-reference) section for more details.
+
+4. Configure the [Sync Generator](/concepts/sync-generators) and CI
+
+Make sure that the `@nx/powerpack-owners:sync-codeowners-file` generators is registered as a `globalGenerator` in `nx.json`:
```

Comment:
We could mention this should have been set by the init generator run by `nx add`?